src/form/stories/FormContainer.js

```diff
@@ -0,0 +1,57 @@
+// @flow
+import React, { Component } from 'react';
+import { DateTime } from 'luxon';
+
+import Form from '..';
+import { schema, uiSchema } from './constants/dataSchemas';
+import {
+  processEntityData,
+  processAssociationEntityData
+} from '../../utils/DataProcessingUtils';
+import { ASSOCIATION_ENTITY_SET_NAMES, ENTITY_SET_NAMES, PROPERTY_TYPE_FQNS } from './constants/mockFQNs';
+import { entitySetIds, propertyTypeIds } from './constants/mockEDM';
+
+import type { EdgeDefinition } from '../../utils/DataProcessingUtils';
+
+const { INCLUDES_ESN } = ASSOCIATION_ENTITY_SET_NAMES;
+const { TASK_LIST_ESN, TASK_ESN } = ENTITY_SET_NAMES;
+const { COMPLETED_DT_FQN } = PROPERTY_TYPE_FQNS;
+
+type Props = {
+  submitAction :(any) => void;
+};
+
+type State = {
+  formData :Object
+};
+
+class FormContainer extends Component<Props, State> {
+
+  getAssociations = () :EdgeDefinition[] => {
+    const taskListEKID :string = '00000000-0000-0000-0000-000000000000';
```

Comment:
What is this?

Reply:
This is just a mock UUID so that it still passes the isValidUUID check that happens within the data utils